### DISCUSSION TITLE
Add zero address

### DISCUSSION THREAD

**Discussion by temptemp3** [reach-sh/reach-lang#1568] [Ideas]
Add a developer I would like to have the zeroAddress available within the context of a reach program to eliminate the need to have to provide the contract the zeroAddress to make it available.

ALGO Zero Address

AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAY5HFKQ
or "0x" + "0".repeat(40);

ETH Zero Address

0x0000000000000000000000000000000000000000
0x0 (null)

**Comment by jeapostrophe** [comment]
Check out my response to this request before: https://github.com/reach-sh/reach-lang/issues/434

**Comment by temptemp3** [comment]
It seems we've been down this path before. Now I remember. Thanks @jeapostrophe

**Comment by temptemp3** [comment]
That was helpful. It helped me realize that there is a better way of doing things that if anyone asked it is the same as having a zero address. Thanks again.

**Comment by temptemp3** [comment]
@jeapostrophe we are good to close this one anytime
